tests/console/commands/test_init.py

```diff
@@ -162,7 +162,8 @@ def test_empty_license(tester):
     ]
     tester.execute(inputs="\n".join(inputs))
 
-    expected = """\
+    python = ".".join(str(c) for c in sys.version_info[:2])
```

Comment:
See above note about using a helper here (I'd even have the helper provide the carat).